src/components/WebPlayBack.tsx

```diff
@@ -78,16 +78,41 @@ function WebPlayback(props: any) {
                 console.log('player toggled');
             }),
             forward : () => player.nextTrack().then(() => {
-                console.log('Skipped to next track!');
-            })
+                console.log('Skipped to next track');
+            }),
+            volume : (event) => player.setVolume(parseInt(event.target.value)/100).then(() => {
+                console.log(parseInt(event.target.value)/100);
```

Comment:
minor: I think we added this for debugging purposes. Can we log a more useful message?

Reply:
I logged a sentence to the console about player volume changing instead. Good idea.